contracts/impls/trustable/WitnetRequestBoardTrustableBase.sol

```diff
@@ -0,0 +1,497 @@
+// SPDX-License-Identifier: MIT
+
+pragma solidity >=0.7.0 <0.9.0;
+pragma experimental ABIEncoderV2;
+
+import "../WitnetRequestBoardUpgradableBase.sol";
+import "../../data/WitnetBoardDataACLs.sol";
+import "../../interfaces/IWitnetRequestBoardAdmin.sol";
+import "../../interfaces/IWitnetRequestBoardAdminACLs.sol";
+import "../../patterns/Payable.sol";
+
+/// @title Witnet Request Board "trustable" base implementation contract.
+/// @notice Contract to bridge requests to Witnet Decentralized Oracle Network.
+/// @dev This contract enables posting requests that Witnet bridges will insert into the Witnet network.
+/// The result of the requests will be posted back to this contract by the bridge nodes too.
+/// @author The Witnet Foundation
+abstract contract WitnetRequestBoardTrustableBase
+    is 
+        Payable,
+        IWitnetRequestBoardAdmin,
+        IWitnetRequestBoardAdminACLs,        
+        WitnetBoardDataACLs,
+        WitnetRequestBoardUpgradableBase        
+{
+    using Witnet for bytes;
+    using Witnet for Witnet.Result;
+
+    uint256 internal constant _ESTIMATED_REPORT_RESULT_GAS = 102496;
+    
+    constructor(bool _upgradable, bytes32 _versionTag, address _currency)
+        Payable(_currency)
+        WitnetRequestBoardUpgradableBase(_upgradable, _versionTag)
+    {}
+
+
+    // ================================================================================================================
+    // --- Overrides 'Upgradable' -------------------------------------------------------------------------------------
+
+    /// Initialize storage-context when invoked as delegatecall. 
+    /// Should fail when trying to initialize same instance more than once.
+    function initialize(bytes memory _initData) virtual external override {
+        address _owner = _state().owner;
+        if (_owner == address(0)) {
+            // set owner if none set yet
+            _owner = msg.sender;
+            _state().owner = _owner;
+        } else {
+            // only owner can initialize:
+            require(msg.sender == _owner, "WitnetRequestBoardTrustableBase: only owner");
+        }        
+
+        if (_state().base != address(0)) {
+            // current implementation cannot be initialized more than once:
+            require(_state().base != base(), "WitnetRequestBoardTrustableBase: already initialized");
+        }        
+        _state().base = base();
+
+        emit Initialized(msg.sender, base(), codehash(), version());
+
+        // Do actual base initialization:
+        setReporters(abi.decode(_initData, (address[])));
+    }
+
+    /// Tells whether provided address could eventually upgrade the contract.
+    function isUpgradableFrom(address _from) external view override returns (bool) {
+        address _owner = _state().owner;
+        return (
+            // false if the WRB is intrinsically not upgradable
+            isUpgradable() && (                
+                _owner == address(0) ||
```

Comment:
You are right. Indeed:

- `_owner == address(0)` was introduced as a way to return `true` in case the actual WRB instance (delegated by a proxy) had not yet been initialized, and because at that time there was no way to transfer owernship.

- As a `WitnetProxy` will never call this method before a new implementation is initialized, and as ownership could now be eventually transferred, indeed this condition MUST be removed.